projects/kubernetes/kubernetes/Makefile

```diff
@@ -5,7 +5,7 @@ RELEASE?=$(shell cat $(BASE_DIRECTORY)/release/$(RELEASE_BRANCH)/$(RELEASE_ENVIR
 ARTIFACT_BUCKET?=my-s3-bucket
 GIT_TAG?=$(shell cat ./$(RELEASE_BRANCH)/GIT_TAG)
 GOLANG_VERSION?=$(shell cat ./$(RELEASE_BRANCH)/GOLANG_VERSION)
-PAUSE_TAG?=$(shell cat ./$(RELEASE_BRANCH)/PAUSE_TAG)
+PAUSE_TAG?=$(shell cat $(wildcard ./$(RELEASE_BRANCH)/PAUSE_*TAG))
```

Comment:
I don't have this command on my mac

Reply:
This is a Make function, so it will be recognized only within the scope of the Makefile.

Reply:
Thanks. Make is so weird